src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -234,6 +234,11 @@ public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatemen
             _sawLocalFunctions = true;
             CheckRefReadOnlySymbols(node.Symbol);
 
+            if (node.Symbol.TypeParameters.Any(typeParameter => typeParameter.HasUnmanagedTypeConstraint))
+            {
+                _factory.CompilationState.ModuleBuilderOpt?.EnsureIsUnmanagedAttributeExists();
+            }
+
```

Comment:
@VSadov this is the merge conflict that I resolved.  I am getting a failed test for Span could you take a look?